### PR TITLE
Adding language switcher to docs main page.

### LINKED ISSUE #885: Add language switcher drop-down to landing page
The other guys (ie http://googlecloudplatform.github.io/gcloud-node/#/) link to us on their landing page, might be nice to return the favor.

### PR DESCRIPTION
**Pull request by dhermes**
Fixes #885.

Can be seen staging at https://blog.bossylobster.com/gcloud-python/#

(Redirected from dhermes.github.io due to https://github.com/dhermes/dhermes.github.io/blob/99a7b13cd657dcf8d9cd90689a35b4c4c199d5a0/CNAME)

### CODE REVIEW

**Comment by dhermes** [comment]
@jgeewax Do you know why the Ruby folks use a PNG instead of SVG image?

**Comment by jgeewax** [comment]
I have no idea. @blowmage : Any idea on this?

**Comment by blowmage** [comment]
The image I was given was for rails, not ruby. I just haven't made a svg for it yet.

**Comment by blowmage** [comment]
I created a SVG for ruby last night. The URL is http://googlecloudplatform.github.io/gcloud-ruby/img/icon-lang-ruby.svg.

**Comment by dhermes** [comment]
Thanks

**Comment by dhermes** [comment]
Updated with the SVG. This PR is ready to go.

File: index.html [hunk 1]
```diff
@@ -149,5 +174,9 @@ <h3 class="block-title">FAQ</h3>
       other Google APIs, the 2 libraries may be used by your application.</p>
       </div>
   </section> <!-- end of FAQ -->
+
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
+    <script>window.jQuery || document.write('<script src="_landing-page-static/js/vendor/jquery-1.10.2.min.js"><\/script>')</script>
```
Comment (tseaver) [inline index.html]: Why two different versions of jQuery?

**Comment by tseaver** [comment]
LGTM except the doubled-load of jQuery.

File: index.html (hunk 1 above)
Comment (dhermes) [inline index.html]: - This is roughly copy-pasta from `ruby` and `node`.
- They are both version `1.10.2`.
- The second only loads a script if `window.jQuery` is not present when the first completes
- The purpose of doing something like this is to use a CDN (ajax.googleapis.com) and fall back to self-hosted JS

**Comment by dhermes** [comment]
Addressed the jQuery question. LMK what you think (feel free to merge if you're OK with it).

**Pull request merged**